frontend/src/NavBar.js

```diff
@@ -29,20 +30,43 @@ const CustomLink = React.forwardRef((props, ref) => (
   />
 ));
 
-export default function NavBar(props) {
+export default function NavBar() {
   const classes = useStyles();
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [logURL, setLogURL] = useState("");
 
+  useEffect(() => {
+    async function fetchLogin() {
+      const result = await getLoginStatus();
+      setIsLoggedIn(result.isLoggedIn);
+      setLogURL(result.logURL);
+    }
+    fetchLogin();
+  }, []);
+  
   return (
     <div className={classes.root}>
       <AppBar position="static">
         <Toolbar className={classes.links}>
           <Typography variant="h6" className={classes.title} component={CustomLink} to="/">
```

Comment:
ooc, what does `forwardRef` do when you create a `CustomLink`?

Reply:
Usually when you encapsulate a component you don't need to pass down refs, but when you do you use `forwardRef`. I think NavLink might need `ref` which is why it's passed down from the encapsulating component `CustomLink`
https://reactjs.org/docs/forwarding-refs.html

Reply:
I see, thanks for the link!